mplc/doc/documentation.md

```diff
@@ -436,6 +436,58 @@ Example: `methods=["Shapley values", "Independent scores", "TMCS"]`
 - `is_quick_demo`: `True` or `False` (default)  
   When set to `True`, the amount of data samples and the number of epochs and mini-batches are significantly reduced, to minimize the duration of the run. This is particularly useful for quick demos or simply for debugging.  
   Example: `is_quick_demo=True`
+  
+## Experiments
+
+At some point of your use of the library you might need to launch several scenarios in a rows. It the same spirit, you might want to repeat one scenario's run, to get rid of the randomness of training, and end with more meaningful results.
+
+The `Experiment` object allows you to set a list of scenarios to run, along with a wanted number of repetition.
+Once instancied, scenarios can be added via the `.add_scenario()`.
+Eventually, all the scenarios can be run, and repeated, by calling the `.run()` method. 
+### Experiment's parameters
+Few parameters can be set:
+    - `experiment_name`: String, default `None`. Name of the experiment, will be used for the save path. if `None` the result will not be saved on disk
```

Comment:
I would not do that. Its the easy way to lose 1 day of computation because you forget to put an `experiment_name`.


I propose:

- if no experiment name is provided, generate one with the current date/time
- Decouple experiment name from the saving --> add a parameter to choose if we want to save the results. default is save=True

Reply:
Also have a mechanism to generate a new experiment name (by adding datetime suffix for instance) if there is already a folder with the same experiment name on disk. I already wrote something like that, I don't know if it's still in the codebase.

Reply:
I agree with @RomainGoussault , except on one slight detail: to me default should be `is_save=False`. Saving results to disk is using a resource. In our world, in 2021, using resources should not be a default behaviour, it should only be done when explicitly needed and requested (so I believe 😎 ).